### PR TITLE
Auto share posts to friends (part 1)

### LINKED ISSUE #159: UX bug for 'View / add recipients'
Version: Latest Chrome Store Ver.

How to reproduce:

Post sth, use 'View / add recipients' functionality and add some recipient.
- The recipient will show up twice during the 'sharing' (following 1st pic)
- The search box UX error - Can't search (2nd pic)

Screen Shot 2019-09-04 at 22 21 56
Screen Shot 2019-09-04 at 22 16 38

### PR DESCRIPTION
**Pull request by Jack-Works**
# Changes
- UI: In PostBox, add all people to selection by default
- Database: Implements methods for UserGroup add/query/delete/modify

This will also fix #159 and #173

### CODE REVIEW

File: config-overrides/index.js [hunk 1]
```diff
@@ -123,7 +124,7 @@ function override(config, env) {
     if (env !== 'development') {
         config.plugins.push(new SSRPlugin('popup.html', src('./src/extension/popup-page/index.tsx')))
         config.plugins.push(new SSRPlugin('index.html', src('./src/index.tsx')))
-        polyfills.map(x => fs.copyFileSync(x, path.join(publicPolyfill, path.basename(x))))
+        polyfills.map(x => void fs.copyFileSync(x, path.join(publicPolyfill, path.basename(x))))
```
Comment (Jack-Works) [inline config-overrides/index.js]: Why add a `void` here?

Comment (SunriseFox) [reply config-overrides/index.js]: It indicates that the return value is not actually used. We could also change it to `forEach` loop, if you like.

File: src/components/shared/SelectPeopleAndGroups/PersonOrGroupInList.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        )
+        const joined = listFormat(nicknamePreviewsForGroup)
+        const groupSize = group.members.length
+        if (groupSize === 0) {
+            secondaryText = geti18nString('person_or_group_in_list_0')
+        } else if (nicknamePreviewsForGroup.length === 0) {
+            secondaryText = geti18nString('person_or_group_in_list_many_no_preview', groupSize + '')
+        } else if (groupSize > nicknamePreviewsForGroup.length) {
+            secondaryText = geti18nString('person_or_group_in_list_many', [joined, groupSize + ''])
+            secondaryText = `${joined}等 共 ${groupSize} 人`
```
Comment (Misaka-0x447f) [inline src/components/shared/SelectPeopleAndGroups/PersonOrGroupInList.tsx]: The value assigned in line 58 was never used.

**Comment by Jack-Works** [comment]
@Misaka-0x447f @SunriseFox Just rebased, need to approve again

**Pull request merged**